src/channels/breakout/index.tsx

```diff
@@ -0,0 +1,303 @@
+			});
+			blocks.current.length = 0;
+
+			if (paddle.current) {
+				paddle.current.destroy();
+				app.current!.stage.removeChild(paddle.current);
+			}
+
+			if (backdrop.current) {
+				backdrop.current.destroy();
+				app.current!.stage.removeChild(backdrop.current);
+			}
+		};
+	});
+
+	// Create initial filters/backdrop
+	useEffect(() => {
+		if (!app.current || crtFilter.current) return;
+		crtFilter.current = new CRTFilter({
+			vignetting: 0.2,
+			time: 0,
+			lineWidth: 1,
+			lineContrast: 0.3,
+			noise: 0.1,
+		});
+
+		app.current.stage.filters = [crtFilter.current];
+
+		backdrop.current = new PIXI.Graphics();
+
+		backdrop.current.beginFill(Color.LIGHT_GRAY);
+		backdrop.current.drawRect(breakout.bounds.x, breakout.bounds.y, breakout.bounds.width, breakout.bounds.height);
+
+		app.current.stage.addChild(backdrop.current);
+	});
+
+	// Create blocks
+	useEffect(() => {
+		if (!app.current || total == null || breakout.blocks.length > 1) return;
+
+		function addBlock(color: Color, position: Position) {
+			const x = position.x;
+			const y = position.y;
+			const width = BLOCK_WIDTH;
+			const height = BLOCK_HEIGHT;
+			const block: Block = {
+				x,
+				y,
+				width,
+				height,
+				collisionBuffer: BALL_RADIUS,
+				color,
+				destroyable: true,
+				destroyed: false,
+			};
+			breakout.addBlock(block);
+			return block;
+		}
+
+		const top = 30;
+		const blocksPerRow = 19;
+		const skipCount = 7;
+		const leftOffset = 26;
+
+		const shouldSkip = (i: number) =>
+			i >= Math.floor((blocksPerRow - skipCount) / 2) && i < Math.floor((blocksPerRow + skipCount) / 2);
+
+		Array.from(
+			{ length: blocksPerRow },
+			(_, i) => !shouldSkip(i) && addBlock(Color.RED, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top }),
+		);
+		Array.from(
+			{ length: blocksPerRow },
+			(_, i) => !shouldSkip(i) && addBlock(Color.ORANGE, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 15 }),
+		);
+		Array.from(
+			{ length: blocksPerRow },
+			(_, i) => !shouldSkip(i) && addBlock(Color.YELLOW, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 30 }),
+		);
+		Array.from({ length: blocksPerRow }, (_, i) =>
+			addBlock(Color.GREEN, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 45 }),
+		);
+		Array.from({ length: blocksPerRow }, (_, i) =>
+			addBlock(Color.BLUE, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 60 }),
+		);
+		Array.from({ length: blocksPerRow }, (_, i) =>
+			addBlock(Color.INDIGO, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 75 }),
+		);
+		Array.from({ length: blocksPerRow }, (_, i) =>
+			addBlock(Color.VIOLET, { x: i * (BLOCK_WIDTH + 5) + leftOffset, y: top + 90 }),
+		);
+	});
+
+	useEffect(() => {
+		if (!app.current || ai.current || !breakout) return;
+
+		ai.current = new BreakoutAI(breakout);
+
+		const aiInterval = setInterval(() => {
+			ai.current?.evaluateMove();
+		}, AI_SAMPLE_MS);
+
+		return () => {
+			clearInterval(aiInterval);
+			ai.current = null;
+		};
+	});
```

Comment:
No dependency array. This will execute everytime this component re-renders.

Reply:
Done.